src/controllers/shared/common.resume.submission.controller.ts

```diff
@@ -110,5 +111,15 @@ const setWebApplicationData = (session: Session, appData: ApplicationData, trans
     mapTrustApiReturnModelToWebModel(appData);
   }
 
+  if (isActiveFeature(config.FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC)) {
+    console.log("\n\n** Remove old NOCs! **\n\n");
+    appData[BeneficialOwnerIndividualKey] = (appData[BeneficialOwnerIndividualKey] as BeneficialOwnerIndividual[])
+      .map( boi => { return { ...boi, [NonLegalFirmNoc]: undefined }; } );
```

Comment:
It won't cause any problems to send a JSON to the API containing undefined values

Reply:
It doesn't seem to but I'm going to look into using that 'delete' function you suggested.

Reply:
Yes, `delete boi[NonLegalFirmNoc]` works too and is clearer I think, so will plan on using that.

Reply:
Updated.